web/mirage/config.ts

```diff
@@ -174,6 +174,45 @@ export default function (mirageConfig) {
         });
       });
 
+      /*************************************************************************
+       *
+       * Project requests
+       *
+       *************************************************************************/
+      // Create a project
+      this.post("/projects", (schema, request) => {
+        let project = schema.projects.create(JSON.parse(request.requestBody));
+        return new Response(200, {}, project.attrs);
+      });
+
+      // Fetch a list of projects.
+      this.get("/projects", () => {
+        const projects = this.schema.projects.all().models;
+        return new Response(200, {}, projects);
+      });
+
+      // Fetch a single project.
+      this.get("/projects/:project_id", (schema, request) => {
+        const project = schema.projects.findBy({
+          id: request.params.project_id,
+        });
+        return new Response(200, {}, project.attrs);
+      });
+
+      // Fetch a product's related resources
```

Comment:
Nit
```suggestion
      // Fetch a project's related resources
```

Reply:
I'm surprised I got the branch name right 😅